invoker.js

```diff
@@ -67,33 +75,47 @@
       return invoker;
     }
 
-    get action() {
+    get command() {
       if (!invokeEventActions.has(this)) {
         throw new TypeError("illegal invocation");
       }
       return invokeEventActions.get(this);
     }
   }
-  makeEnumerable(InvokeEvent.prototype, "invoker");
-  makeEnumerable(InvokeEvent.prototype, "action");
+  enumerate(CommandEvent.prototype, "invoker");
```

Comment:
Is this still called invokers?

Reply:
The property on the event is still called `invoker`. But `action` should change to `command`.